frontend/src/views/CampCreate.vue

```diff
@@ -19,7 +19,7 @@
             <e-text-field
               v-model="camp.motto"
               :name="$tc('entity.camp.fields.motto')" />
-            <!--<e-select
+            <e-select
```

Comment:
Im dem select werden im Moment alle Camps angezeigt, nicht nur die Prototypes. Falls kein Quickfix, kann das von mir aus aber auch in ein separates Issue.

Reply:
Das DropDown habe ich wieder eingeschalten, damit ich ein Prototype an die API senden kann.
Der Prototype-Filter auf der Camp-Liste gibt es aber noch nicht wieder.
Issue erfasst: #2362